api-server/common/models/User-Identity.js

```diff
@@ -43,7 +43,7 @@ export default function(UserIdent) {
     // get the email from the auth0 (its expected from social providers)
     const email =
       profile && profile.emails && profile.emails[0]
-        ? profile.emails[0].value
+        ? ('' + profile.emails[0].value).toLowerCase()
```

Comment:
Trouble is, if the value is `null` then `email` ends up being the string `'null'` and I think it should be `''`.

Instead, I'd check if `profile.emails[0].value` exists and is a string.  If that's the case, then return the lower case version, otherwise just return the empty string.

Reply:
Right. I have tried to implement something encompassing enough.